src/message/fetchActions.js

```diff
@@ -82,7 +82,7 @@ export const fetchMessages = (
 };
 
 export const fetchMessagesAtFirstUnread = (narrow: Narrow): Action =>
-  fetchMessages(narrow, 0, config.messagesPerRequest, 0, true);
+  fetchMessages(narrow, 0, config.messagesPerRequest / 4, 3 * config.messagesPerRequest / 4, true);
```

Comment:
or may be
`fetchMessages(narrow, 0, config.messagesPerRequest / 2, config.messagesPerRequest / 2, true);`

but `1:3` will be better as already read message are not so important and we should not send time in loading them.